hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMBlockDeletingService.java

```diff
@@ -225,7 +235,16 @@ public void notifyStatusChanged() {
   public boolean shouldRun() {
     serviceLock.lock();
     try {
-      return serviceStatus == ServiceStatus.RUNNING;
+      long alreadyWaitTimeInMillis = clock.millis() - lastTimeToBeReadyInMillis;
```

Comment:
Could we rename:

lastTimeToBeReadyInMillis -> safemodeExitMillis
waitTimeInMillis -> safemodeExitRunDelayMillis